pkg/apis/migration/v1alpha1/migstorage_types.go

```diff
@@ -195,6 +195,21 @@ func (r *MigStorage) GetBSL(client k8sclient.Client) (*velero.BackupStorageLocat
 	return nil, nil
 }
 
+// Default the VSL settings (when not defined) using the BSL settings.
+func (r *MigStorage) DefaultVSLSettings() {
+	if r.Spec.VolumeSnapshotProvider != "" {
+		return
+	}
+	r.Spec.VolumeSnapshotProvider = r.Spec.BackupStorageProvider
+	switch r.Spec.BackupStorageProvider {
+	case "aws":
```

Comment:
@jortel this seems really handy! 

A few things I'm thinking about around this:

 - Would the relationship go the other way as well? Would undefined BSL settings be filled in by whatever the user supplied for VSL? If not, why only make this work one way?

 - Is it ever dangerous to assume that the user _wants_ us to use the same AWS credentials for Volume Snapshotting? I can't think of a good reason why someone wouldn't want to use the same credentials... Maybe just something to keep in mind.

Reply:
Since the BSL is required there really isn't a use case for defaulting the BSL settings using the VSL settings.  The decision to use the BSL settings for the VSL (when not defined) is the result of a slack discussion with Dylan and Scott.  Is it safe?  The consensus was yes.

Reply:
ACK